### ISSUE TITLE
加入 computer vision 和 NLP 参考文献引用

### ISSUE THREAD

**Issue by heroxbd** [heroxbd/waveform-analysis#23]
Advances in neural networks have brought breakthroughs in various domains like computer vision [CITE!] and natural language processing[CITE!].

**Comment by TakaraResearch** [comment]
ComputerVision(机器视觉)
图像分类
1. Deep residual learning for image recognition                              
He, Kaiming, et al. "Deep residual learning for image recognition." Proceedings of the IEEE conference on computer vision and pattern recognition. 2016.
目标检测
2. Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks
Ren, Shaoqing, et al. "Faster R-CNN: towards real-time object detection with region proposal networks." IEEE transactions on pattern analysis and machine intelligence 39.6 (2016): 1137-1149.
目标分割
3.Mask r-cnn
He, Kaiming, et al. "Mask r-cnn." Proceedings of the IEEE international conference on computer vision. 2017.

NLP (自然语言处理)
（机器翻译 等 Transformer结构）
Vaswani, Ashish, et al. "Attention is all you need." arXiv preprint arXiv:1706.03762 (2017).
语言模型 Bert
Devlin, Jacob, et al. "Bert: Pre-training of deep bidirectional transformers for language understanding." arXiv preprint arXiv:1810.04805 (2018).
语言模型 GPT-3
Brown, Tom B., et al. "Language models are few-shot learners." arXiv preprint arXiv:2005.14165 (2020).

**Comment by TakaraResearch** [comment]
ComputerVision(机器视觉)
图像分类
1. Deep residual learning for image recognition                              
He, Kaiming, et al. "Deep residual learning for image recognition." Proceedings of the IEEE conference on computer vision and pattern recognition. 2016.
目标检测
2. Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks
Ren, Shaoqing, et al. "Faster R-CNN: towards real-time object detection with region proposal networks." IEEE transactions on pattern analysis and machine intelligence 39.6 (2016): 1137-1149.
目标分割
3. Mask r-cnn
He, Kaiming, et al. "Mask r-cnn." Proceedings of the IEEE international conference on computer vision. 2017.

NLP (自然语言处理)
（机器翻译 等 Transformer结构）
1. Vaswani, Ashish, et al. "Attention is all you need." arXiv preprint arXiv:1706.03762 (2017).
语言模型 Bert
2. Devlin, Jacob, et al. "Bert: Pre-training of deep bidirectional transformers for language understanding." arXiv preprint arXiv:1810.04805 (2018).
语言模型 GPT-3
3. Brown, Tom B., et al. "Language models are few-shot learners." arXiv preprint arXiv:2005.14165 (2020).

**Comment by TakaraResearch** [comment]
ComputerVision(机器视觉)
1.图像分类
Deep residual learning for image recognition                              
He, Kaiming, et al. "Deep residual learning for image recognition." Proceedings of the IEEE conference on computer vision and pattern recognition. 2016.
2.目标检测
Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks
Ren, Shaoqing, et al. "Faster R-CNN: towards real-time object detection with region proposal networks." IEEE transactions on pattern analysis and machine intelligence 39.6 (2016): 1137-1149.
3.目标分割
Mask r-cnn
He, Kaiming, et al. "Mask r-cnn." Proceedings of the IEEE international conference on computer vision. 2017.

NLP (自然语言处理)
1.机器翻译 等 Transformer结构
Vaswani, Ashish, et al. "Attention is all you need." arXiv preprint arXiv:1706.03762 (2017).
2.语言模型 Bert
Devlin, Jacob, et al. "Bert: Pre-training of deep bidirectional transformers for language understanding." arXiv preprint arXiv:1810.04805 (2018).
3.语言模型 GPT-3
Brown, Tom B., et al. "Language models are few-shot learners." arXiv preprint arXiv:2005.14165 (2020).
